frontend/src/scenes/experiments/utils.ts

```diff
@@ -97,7 +97,7 @@ export function getMinimumDetectableEffect(
         const targetConversionRate = Math.round(currentConversionRate * (1 + targetConversionRateIncrease))
         const mde = Math.ceil(targetConversionRate - currentConversionRate)
 
-        return mde || 5
+        return mde || 30
```

Comment:
Just a side note for later: I think we can get rid of this whole function actually. It think this calculation tries to estimate an absolute MDE, but we should just default to 30% relative MDE. So no need for this logic. But that's another PR.

Reply:
added to my todo list 😉